composer-require-checker.json

```diff
@@ -0,0 +1,19 @@
+{
+    "symbol-whitelist" : [
+        "null", "true", "false",
+        "static", "self", "parent",
+        "array", "string", "int", "float", "bool", "iterable", "callable", "void", "object", "mixed", "never"
+    ],
+    "php-core-extensions" : [
```

Comment:
Aren't these all included out-of-the-box with @maglnet /composer-require-checker (and no config)?

Reply:
Apparently there can be only 1 config file - CI depends on it being here so you have to copy paste the defaults: https://github.com/maglnet/ComposerRequireChecker#configuration

Reply:
@maglnet is this the expected modus operandi of that package? Feels very broken to me - perhaps we should adjust it?

Reply:
@Ocramius @gsteel 
It should not be necessary to add the defaults. Maybe there's an issue when running it as a dev dependency instead of a phar file.
I'll check this.

Reply:
In the case, the presence of the file is what triggers the ci run so according to the docs, the config file would have to duplicate the defaults. Right?

Reply:
Ah, now I got it, you added the part to `scan-files`. May you just try to only add the `scan-files` part you want to adjust?

As far as I see, composer-require-checker only loads this section, but if you modify a section you have to completely add the section, maybe that's a bit confusing within the readme.

Reply:
To be more preciese, try it with the following config-file:
```json
{
    "scan-files" : [
        "bin/*.php"
    ]
}
```

Reply:
Thanks @maglnet - That did it!

Reply:
Excellent, thanks @maglnet!